testsuite/features/secondary/min_salt_formulas_advanced.feature

```diff
@@ -99,6 +99,7 @@ Feature: Use advanced features of Salt formulas
      And I click on "Clear values" and confirm
      And I click on "Save Formula"
      Then I should see a "Formula saved" text
+     When I refresh the pillar data
      And the pillar data for "testing:str" should be "" on "sle_minion"
```

Comment:
`Then` maybe

Reply:
Ok, I will change in all of the steps bellow the refresh.